cognitedata/reveal
# chore: better dark theme logo

#### Type of change

![Chore](https://img.shields.io/badge/Type-Chore-lightgrey) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/

## Description :pencil:


New dark theme Cognite logo, as described in https://www.stefanjudis.com/notes/how-to-define-dark-light-mode-images-in-github-markdown/#github-rolls-out-the-%60picture%60-element (Third paragraph)

## How has this been tested? :mag:




## Test instructions :information_source:



- Go to "Files changed"
- Click the dots on the top right of "README.md"
- Click "View File"
- Notice that (if you're using dark mode on Github) the Cognite logo is now white

## Checklist :ballot_box_with_check:
